packages/fetch-engine/src/downloadZip.ts

```diff
@@ -48,7 +56,7 @@ export async function downloadZip(
 ): Promise<DownloadResult> {
   const tmpDir = tempy.directory()
   const partial = path.join(tmpDir, 'partial')
-  const { sha256, zippedSha256 } = await fetchSha256(url)
+  const [zippedSha256, sha256] = await Promise.all([fetchChecksum(url), fetchChecksum(url.slice(0, url.length - 3))])
```

Comment:
These keys had their order reversed. Would it make sense to stick `sha256` back as the first item in the array and adjust the right-hand side accordingly?

Reply:
I don't think it matters here, this code is correct, I don't like the `url.slice` so much but that was there before 😅